### PR TITLE
Fix diff editor options

### LINKED ISSUE #255: Configuration service now works correctly with diffEditor.renderSideBySide
Change of diffEditor.renderSideBySide does not works [here](https://github.com/CodinGame/vscode/blob/f63107edad005ed0ffe46a9ce42ac5f0fdafa4d2/src/vs/workbench/browser/parts/editor/editorCommands.ts#L408)

### PR DESCRIPTION
**Pull request by CGNonofr**
supposed to fix #255 but maybe I misunderstood and fix something else?

### CODE REVIEW

**Comment by CGNonofr** [comment]
> @CGNonofr no more pre-releases?

Are you saying I'm too lazy ?  ^^

**Comment by kaisalmen** [comment]
> Are you saying I'm too lazy ? ^^

Exactly 😆 No, just saw that the last one was quite a while ago. Is it easy to produce (like press a button)? Otherwise I need to come up with a local dev setup for `monaco-languageclient` (script with auto-switching to linked deps)

**Comment by CGNonofr** [comment]
> > Are you saying I'm too lazy ? ^^
> 
> Exactly 😆 No, just saw that the last one was quite a while ago. Is it easy to produce (like press a button)? Otherwise I need to come up with a local dev setup for `monaco-languageclient` (script with auto-switching to linked deps)

Just a command to run, I should add a manual task in the CI!

In progress :)

**Comment by kaisalmen** [comment]
> In progress :)

Thank you

**Comment by CGNonofr** [comment]
> > In progress :)
> 
> Thank you

Done :)

**Comment by kaisalmen** [comment]
@CGNonofr just a heads up. I won't be able to fully test this tonight, because I have to test/implement the other changes as well. I will continue tomorrow.

**Comment by kaisalmen** [comment]
`monaco-languageclient` builds we can ship it! 🙈

Jokes aside, I need to test this, especially the side-by-side diff. We use it one level up

File: src/service-override/extensionGallery.ts [hunk 1]
```diff
@@ -27,6 +27,12 @@ import { ExtensionEnablementService } from 'vs/workbench/services/extensionManag
 import { IInstantiationService, ILabelService, IRemoteAgentService } from '../services'
 import 'vs/workbench/contrib/extensions/browser/extensions.contribution'
 import 'vs/workbench/contrib/extensions/browser/extensions.web.contribution'
+import { registerAssets } from '../assets'
+
+// plugin-import-meta-asset only allows relative paths
+registerAssets({
```
Comment (CompuIves) [inline src/service-override/extensionGallery.ts:33]: Nice side change :)

**Comment by CompuIves** [comment]
So this only applies to the standalone diff editor, right?

**Comment by CGNonofr** [comment]
> So this only applies to the standalone diff editor, right?

Correct!

**Comment by CGNonofr** [comment]
@kaisalmen as it only includes fixes not closely related to the api (thus monaco-languageclient) and since it fixes the demo build/deploy, what about merging it (it you see no issues with the code iself OFC) and fixing any issues in a next version?

**Comment by kaisalmen** [comment]
@CGNonofr yes, go ahead. Sorry, didn't get to it today, yet. If something comes up regarding mlc (Jump from 1.83.5 to 1.83.10), I let you know / raise an issue / PR.

**Review by kaisalmen — APPROVED** [review]
Go ahead.

**Pull request merged**